src/components/Property.astro

```diff
@@ -1,5 +1,5 @@
-
 ---
+/* eslint-disable tailwindcss/no-custom-classname */
```

Comment:
This is because the `expanded` class is used to open and close those attributes. Have left it for now but might cull it at a later date 👀